tests/pmDownloadTest.php

```diff
@@ -18,8 +18,8 @@ public function testPmDownload() {
   // @todo Test pure drush commandfile projects. They get special destination.
   public function testDestination() {
     // TODO: enable test when devel available for Drupal 9
-    if (UNISH_DRUPAL_MAJOR_VERSION >= 9) {
-      $this->markTestSkipped('Drupal 9 version of devel not available yet.');
+    if (version_compare(UNISH_DRUPAL_MAJOR_VERSION . str_replace('.x-dev', '.0-dev', UNISH_DRUPAL_MINOR_VERSION), '8.9.0-dev', '>=')) {
+      $this->markTestSkipped('Drupal 8.9 / 9.0 version of devel not available yet.');
```

Comment:
The 8.3.x version of devel works with d9.

Reply:
Cool.